packages/odata-service-inquirer/src/types.ts

```diff
@@ -202,40 +202,6 @@ export type EntityRelatedAnswers = EntitySelectionAnswers &
     AnnotationGenerationAnswers &
     AlpTableConfigAnswers;
 
-export type CapRuntime = 'Node.js' | 'Java';
```

Comment:
In retrospect, we should be re-exporting this type from its new location if its needed by the APIs here.